src/Annotations/Serializer/HypothesisClientAnnotationNormalizer.php

```diff
@@ -4,13 +4,15 @@
 
 use eLife\ApiSdk\ApiSdk;
 use eLife\HypothesisClient\Model\Annotation;
+use HTMLPurifier;
 use League\CommonMark\Block\Element;
 use League\CommonMark\DocParser;
 use League\CommonMark\ElementRendererInterface;
 use Psr\Log\LoggerInterface;
 use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
 use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use function eLife\Annotations\Serializer\CommonMark\escape_math;
 
 final class HypothesisClientAnnotationNormalizer implements NormalizerInterface, NormalizerAwareInterface
```

Comment:
`NormalizerAwareInterface` not needed anymore?

Reply:
Will remove